openmetadata-ui/src/main/resources/ui/src/components/BotDetails/AuthMechanismForm.tsx

```diff
@@ -213,8 +219,6 @@ const AuthMechanismForm: FC<Props> = ({
       }
     } catch (error) {
       showErrorToast(error as AxiosError);
-    } finally {
-      setIsConfirmationModalOpen(false);
```

Comment:
Can you explain why we removed this and why we are not marking it as false anywhere else

Reply:
@chirag-madlani  we are making it in handleBotUpdate in try block on line number 179.